packages/perseus/src/widgets/interactive-graphs/grid.tsx

```diff
@@ -8,12 +8,18 @@ import type {GraphRange} from "../../perseus-types";
 import type {SizeClass} from "../../util/sizing-utils";
 import type {vec} from "mafs";
 
+/*
+gridStep: Where the lines on the grid will render.
+tickStep: Where the little black axis lines & labels (ticks) should render. (e.g. 1, 2, 3, 4, 5)
+snapStep: Where the graph points will lock to when they are dragged.
+*/
```

Comment:
I added these comments for my own sanity, and wondered if I should keep them. Happy to delete this if it's not useful however.

Reply:
I love that these exist, please don't remove them entirely. 

But, could these go on the `GridProps` type itself?  Or some other type where these keys are defined that we could document?